flask_rebar/rebar.py

```diff
@@ -15,11 +15,14 @@
 from collections import namedtuple
 from copy import copy
 from functools import wraps
+from packaging import version
```

Comment:
For some reason, i need to install `packaging` specifically using pip3 in python 3.5

Reply:
Ah, crap. I should've used another way of doing this comparison

Reply:
Let me change this real quick

Reply:
https://github.com/plangrid/flask-rebar/pull/29